api/application.ts

```diff
@@ -274,7 +275,7 @@ app.get('/external/ping', (req, res) => {
 /**
  * We are attaching authentication to all subsequent routes.
  */
-// app.use(auth.attach) // its called in routes.ts - no need to call it here
+app.use(attach) // its called in routes.ts - no need to call it here
```

Comment:
we don't have an `attach` method in auth/index.ts anymore? I commented all that out in favour of initialising our lib